app/actions/remote/post.ts

```diff
@@ -552,3 +552,38 @@ export const togglePinPost = async (serverUrl: string, postId: string) => {
         return {error};
     }
 };
+
+export const markPostAsUnread = async (serverUrl: string, postId: string, channelId: string) => {
```

Comment:
since we already have a postId, we can get the channel from it and then access myChannel, there is no need for the `channelId` argument